src/Libraries/CoreNodeModels/CreateList.cs

```diff
@@ -7,17 +7,18 @@
 
 namespace DSCoreNodesUI
 {
-    [NodeName("List.Create")]
-    [NodeDescription("ListCreateDescription", typeof(DSCoreNodesUI.Properties.Resources))]
-    [NodeSearchTags("ListCreateSearchTags", typeof(DSCoreNodesUI.Properties.Resources))]
+    [NodeName("List.ByItems")]
+    [NodeDescription("ListByItemsDescription", typeof(DSCoreNodesUI.Properties.Resources))]
+    [NodeSearchTags("ListByItemsSearchTags", typeof(DSCoreNodesUI.Properties.Resources))]
     [NodeCategory(BuiltinNodeCategories.CORE_LISTS_CREATE)]
     [IsDesignScriptCompatible]
+    [AlsoKnownAs("List.Create")]
```

Comment:
We don't need to write migration, because in fact nothing changed. `[NodeName("List.Create")]` is Nickname. Node type didn't change.